docs/src/main/asciidoc/funqy-amazon-lambda-http.adoc

```diff
@@ -8,14 +8,21 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-If you like link:funqy-http[Funqy HTTP], you can use it on AWS Lambda.  Quarkus allows you to expose multiple
-Funqy functions through HTTP deployed as one AWS Lambda.
+If you want to allow HTTP clients to invoke on your Funqy functions on AWS Lambda, Quarkus allows you to expose multiple
+Funqy functions through HTTP deployed as one AWS Lambda.  This approach does add overhead over the
+regular Funqy AWS Lambda integration and also requires you to use AWS API Gateway.
 
 include::./status-include.adoc[]
 
 Follow the link:amazon-lambda-http[Amazon Lambda Http Guide].  It walks through using a variety of HTTP
 frameworks on Amazon Lambda, including Funqy.
 
+WARNING: The Funqy HTTP + AWS Lambda binding is not a replacement for REST over HTTP.  Because Funqy
```

Comment:
Small suggestion; I'd put this into an admonition block (here and the other occurrences):

```
[NOTE] 
====
The Funqy HTTP + AWS Lambda binding...
====

Reply:
Well, it is an admonition, it just uses the shortcut. I think it's good enough this way.